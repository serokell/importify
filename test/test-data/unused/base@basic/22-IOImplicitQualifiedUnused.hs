module IOImplicitQualifiedUnused where

import System.IO as IO (putStrLn)

main :: IO ()
main = pure ()
