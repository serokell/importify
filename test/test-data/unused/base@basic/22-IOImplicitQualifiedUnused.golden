module IOImplicitQualifiedUnused where


main :: IO ()
main = pure ()
